### PR TITLE
Update travis to test ruby 2.2

### PR DESCRIPTION
**Pull request by seuros**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,11 +2,12 @@
 language: ruby
 
 rvm:
-  - 1.8.7
-  - 1.9.2
-  - 1.9.3
+  - 2.1
+  - 2.2
```
Comment (vipulnsward) [inline .travis.yml]: These need to be re-arranged too, just like below.

Comment (seuros) [inline .travis.yml]: Travis run the tests as we define them: 
I arranged them by % of their usage 
```
- 2.1
- 2.2
- 2.0.0
- 1.9.3
- 1.9.2
- 1.8.7
```

Comment (rkh) [inline .travis.yml]: Travis CI does not necessarily start jobs in order.

Comment (seuros) [inline .travis.yml]: Oh. Thank you for this information.

**Pull request merged**